astacus/common/compatibility.py

```diff
@@ -0,0 +1,24 @@
+"""
+Copyright (c) 2022 Aiven Ltd
+See LICENSE for details
+"""
+
+__all__ = ("to_thread",)
```

Comment:
TBH this isn't really desirable to me - I have had in my Cassandra dev branch 3.9 only already for a bit, and I think we want to go there rather than add workarounds (3.9 isn't super new either). I made PR out of that now: https://github.com/aiven/astacus/pull/80